### PR TITLE
project-business-site Emma

### PR DESCRIPTION
**Pull request by emmapre**
This is my finished project. 
I'm quite happy with how it turned out, I managed to make everything that I wanted.
I tried to add an animation for the border when focusing the inputs but it didn't work the way I wanted. I wanted it to become solid from right to left but I only succeeded in transitioning it from thin to thicker. 
I think there are some repitions, the form-container and the accordion-container are pretty much styled the same so they could have the same class. I also think there are som doubles in the media queries that don't need to be there.

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Hi! Could you add your netlify link in this pull request or in a readme of your project? Thanks!

**Comment by emmapre** [comment]
Here is the link! https://infallible-cray-86c205.netlify.com/

File: code/script.js [hunk 1]
```diff
@@ -7,3 +7,9 @@ function toggle() {
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
+document.getElementById("section5").onclick = toggle
+
+// This could probably be made smarter but right now I don't know more js :)
```
Comment (ericaglimsholt) [inline code/script.js]: This is great for now! 👏

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,286 @@
...
+
+/* *********CONTAINER********** */
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items:center;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+    margin-bottom: 100px;
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
  margin-bottom: 100px;
```

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,286 @@
...
+  margin-top: 20px;
+  height:30px;
+  font-family: Quattrocento;
+} 
+
+.radio {
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center; 
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,286 @@
...
+  button:hover {
+    box-shadow: none;
+  }
+
+  button:focus {
+    box-shadow: inset 5px 5px 2px 0px rgba(0, 0, 0, 0.45);
+    -webkit-box-shadow: inset 5px 5px 2px 0px rgba(0, 0, 0, 0.45);
+    -moz-box-shadow: inset 5px 5px 2px 0px rgba(0, 0, 0, 0.45);
+    transition: all 0.4s ease 0s;
+    }
```
Comment (ericaglimsholt) [inline code/style.css]: Unindent one step

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,286 @@
...
+  font-family: Quattrocento;
+  font-weight: bold;
+  background: #d7c79e;
+  padding: 20px;
+  border-radius: 50px;
+  display: inline-block;
+  border: none;
+  transition: all 0.4s ease 0s;
+  box-shadow: 5px 5px 2px 0px rgba(0, 0, 0, 0.45);
+  }
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
}
```

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,286 @@
...
+    min-height: 700px;
+  }
+
+
+}
+
+@media screen and (min-width: 900px) { 
+h1 { 
+  font-size: 75px;
+}
```
Comment (ericaglimsholt) [inline code/style.css]: Indent one step!

**Pull request closed without merging**